miner/src/pool/queue.rs

```diff
@@ -32,7 +33,7 @@ use pool::{
 };
 use pool::local_transactions::LocalTransactionsList;
 
-type Listener = (LocalTransactionsList, (listener::Notifier, listener::Logger));
+type Listener = (LocalTransactionsList, (listener::Notifier, (listener::Logger, listener::TransactionsPoolNotifier)));
```

Comment:
That starts to look pretty ugly, we could have a macro in `parity-common` to implement `Listener` for tuples of different sizes.
(Note it's not strictly related to this PR)